samples/trace/index.js

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// [START opentelemetry_trace_samples]
+'use strict';
+
+// [START opentelemetry_trace_import]
+const opentelemetry = require('@opentelemetry/api');
+const {NodeTracerProvider} = require('@opentelemetry/node');
+const {SimpleSpanProcessor} = require('@opentelemetry/tracing');
+const {
+  StackdriverTraceExporter,
+} = require('@google-cloud/opentelemetry-cloud-trace-exporter');
+// [END opentelemetry_trace_import]
+
+// [START setup_exporter]
```

Comment:
In Go examples the requirement was that these labels have same prefix. So the above should be changed to
`opentelemetry_trace_setup_expoter`

Reply:
@willgoldby could you please confirm?

Reply:
We did do that for the Go samples, but I'm not sure if it's a general rule. I know in other places it hasn't been required, so it looks like each repository has their own standards.